cmake/BuildicubWalkingDCM.cmake

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2018  iCub Facility, Istituto Italiano di Tecnologia
+# Authors: Stefano Dafarra <stefano.dafarra@iit.it>
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+include(FindOrBuildPackage)
+
+find_or_build_package(YARP QUIET)
+find_or_build_package(iDynTree QUIET)
+find_or_build_package(UnicyclePlanner QUIET)
+find_or_build_package(OsqpEigen QUIET)
+find_or_build_package(qpOASES QUIET)
+
+set(icubWalkingDCM_DEPENDS "")
+list(APPEND icubWalkingDCM_DEPENDS YARP)
+list(APPEND icubWalkingDCM_DEPENDS iDynTree)
+list(APPEND icubWalkingDCM_DEPENDS UnicyclePlanner)
+list(APPEND icubWalkingDCM_DEPENDS OsqpEigen)
+list(APPEND icubWalkingDCM_DEPENDS qpOASES)
+
+ycm_ep_helper(icubWalkingDCM TYPE GIT
```

Comment:
@GiulioRomualdi  Can we agree on a better name? See https://github.com/robotology/capture-point-walking/blob/master/code/cpp/CMakeLists.txt#L7 for the original definition of this name. Having a `package` name that is completely different from the repository name is quite misleading. If necessary, we can still rename the repo.